assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -100,6 +100,9 @@ const doFailureBuffer = (
   }
 };
 
+const isSuccess = (response: ApiResponse) =>
+  response != null && response.state === "success";
```

Comment:
I was thinking this helper func would return true for both forms of successful response: `success` and `simulation_success`. I might be missing a reason why we shouldn't lump them together like that, though.

If that does make sense to you, this also needs to check for `"simulation_success"`—either `["success", "simulation_success"].includes(response.state)` or `(response.state === "success" || response.state === "simulation_success")`